dasharo-performance/cpu-frequency.robot

```diff
@@ -341,11 +341,14 @@ CPU Runs On Expected Frequency (Ubuntu)
         Log To Console    \n ----------------------------------------------------------------
         Log To Console    ${timer} min.
         @{frequencies}=    Get CPU Frequencies In Ubuntu
+        ${cpu_max_frequency_tol}=    Evaluate    ${CPU_MAX_FREQUENCY} * 1.125
+        ${cpu_min_frequency_tol}=    Evaluate    ${CPU_MIN_FREQUENCY} * 0.875
```

Comment:
Why calculating this in loop over and over, it won't change.

Reply:
oh wait there are 2 loops, my bad